accounts-cluster-bench/src/main.rs

```diff
@@ -683,6 +720,8 @@ fn run_accounts_bench(
             let _ = executor.drain_cleared();
         }
 
+        start_bench_tx.send(()).unwrap();
```

Comment:
won't this only unblock one of the bench threads?

For this kind of stuff, I think
https://doc.rust-lang.org/stable/std/sync/struct.Barrier.html would be more idiomatic

Reply:
Ah, looks like stacking got me already 😅

Reply:
> won't this only unblock one of the bench threads?

Dude, I don't know how computers work.

ty for the pointer.

Reply:
@nickfrosty, this is – in my opinion – a perfect example of what we were talking about earlier today. In this particular instance I was on the ‘don't know what I don't know‘ side of the house. I needed a _guide_ on synchronization primitives. I spent a ton of time with the Rust API docs, but they didn't lead me to the right solution. Furthermore, as soon as Alessandro gave me the pointer to `Barrier`, I no longer had any _use_ for the web-based API docs; I found all the answers I needed in the Rust LSP / the code itself.